webapp/settings.py

```diff
@@ -112,6 +110,10 @@
 
 ASSET_SERVER_URL = "https://assets.ubuntu.com/v1/"
 
+DATABASES = {
```

Comment:
why adding this?

Reply:
Yeah, it's a strong requirement that this site doesn't have a database.

Reply:
What do you think about the testing approach here?

Since we have no DB I have found the following approach on [Stackoverflow](https://stackoverflow.com/questions/5917587/django-unit-tests-without-a-db/):

>  from django.test.runner import DiscoverRunner
> 
>  class NoDbTestRunner(DiscoverRunner):
>    """ A test runner to test without database creation/deletion """
> 
>    def setup_databases(self, **kwargs):
>      pass
> 
>    def teardown_databases(self, old_config, **kwargs):
>      pass
> 
> Use like that :
> 
> python manage.py test app --testrunner=app.filename.NoDbTestRunner
> 

The only problem here is that the tests have to be called with explicitly specifying the test runner. Can still easily be called in the `package.json` of course.

Reply:
I think you can set the test_runner in settings.py!